vault/expiration.go

```diff
@@ -54,15 +52,23 @@ const (
 // the ExpirationManager will handle doing automatic revocation.
 type ExpirationManager struct {
 	router     *Router
+	topView    *BarrierView
```

Comment:
This doesn't seem to be used anywhere?

Reply:
It was used in some tests, although I think those tests are now gone.